### DISCUSSION TITLE
404: NOT_FOUND Code: NOT_FOUND ID: bom1::hdvzt-1717714023080-74e3b0c215cf

### DISCUSSION THREAD

**Discussion by zaindunnenterprisesinc** [vercel/community#7115] [Bug Report] [answered]
### Page to Investigate

www.dunnenterprisesinc.com

### Steps to Reproduce

404: NOT_FOUND Code: NOT_FOUND ID: bom1::hdvzt-1717714023080-74e3b0c215cf

 i am getting the same error can someone please help me with this?

**Comment by zaindunnenterprisesinc** [comment]
@rauchg @tomocchino @TooTallNate @vvo any one please help me out with this thing?

**Comment by amyegan** [comment, marked as answer]
Hi @zaindunnenterprisesinc. Thanks for sharing the domain. It looks like your site doesn't have an `index`. I see "Hello World!" if I visit https://www.dunnenterprisesinc.com/api though.

There are a lot of possible causes that depend on the project framework, project settings, and your workflow. But I shared some tips in another post that can help you find the cause - https://github.com/vercel/community/discussions/6980#discussioncomment-9705033
